test/dsl_test.go

```diff
@@ -232,7 +265,7 @@ func (o *opt) failf(format string, objs ...interface{}) {
 
 func (o *opt) expectSuccess(reason string, err error) {
 	if err != nil {
-		o.t.Fatalf("Error: %s: %v", reason, err)
+		o.t.Errorf("Error: %s: %v", reason, err)
```

Comment:
Ah, because of the new `parallel` thing.  Maybe you can add an indicator on an `opt` that this is being executed in a different goroutine, and only use `Errorf` in that case?  I'd rather not change the existing behavior too much.  It makes the test output harder to decipher when the test continues past a fatal error.